recipes/gplearn/LICENSE.txt

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2015, Trevor Stephens
```

Comment:
Both the github archive and the pypi source distribution have the license file included in them, so you should remove this file

Reply:
Oh okay, I'm quite new to creating conda-forge recipes and I saw the comment in the example meta.yaml that said:

*It is strongly encouraged to include a license file in the package*

Happy to remove it if that is okay... do I still link back to one of the original copies of the license or do I drop the license_file entry completely?

Reply:
If I do remove the LICENSE.txt file please let me know how to update the meta.yaml - or if it's easier refer me to another feedstock that does the license the way you prefer.

Cheers,
Owen

Reply:
Sorry I missed your other comment - changed as requested I hope.